Documentation/CodingGuidelines/PhpFileFormatting/PhpSyntaxFormatting.rst

```diff
@@ -406,31 +406,31 @@ PHP features
 The use of the newest PHP features is strongly recommended for
 extensions and mandatory for the TYPO3 core.
 
-Class functions must have access type specifiers: :code:`public`,
-:code:`protected` or :code:`private`. Notice that :code:`private` may
-prevent XCLASSing of the class. Therefore :code:`private` can be used
+Class functions must have access type specifiers: :php:`public`,
+:php:`protected` or :php:`private`. Notice that :php:`private` may
+prevent XCLASSing of the class. Therefore :php:`private` can be used
 only if it is absolutely necessary.
 
-Class variables must use access specifiers instead of the :code:`var`
+Class variables must use access specifiers instead of the :php:`var`
 keyword.
 
-Type hinting must be used when the function expects an :code:`array` or
-an :code:`instance` of a certain class. Example::
+Type hinting must be used when the function expects an :php:`array` or
+an :php:`instance` of a certain class. Example::
```

Comment:
Please drop the code block completely here, it's a regular word in this sentence.